### PR TITLE
Setup an app skeleton

### PR DESCRIPTION
**Pull request by erykpiast**

### CODE REVIEW

File: src/Main.elm [hunk 1]
```diff
@@ -0,0 +1,25 @@
+module Main exposing (main)
+
+import Browser
+import Html exposing (button, div, text)
+import Html.Events exposing (onClick)
+
+main =
+  Browser.sandbox { init = 0, update = update, view = view }
```
Comment (adrianmroz) [inline src/Main.elm:8]: We’re grown ups, we can use `Browser.Application`

**Review by adrianmroz — APPROVED** [review]

File: src/Main.elm (hunk 1 above)
Comment (erykpiast) [reply src/Main.elm:8]: Sure we can! But until we need all those things like URL change callbacks, a sandbox is enough, isn't it?

Comment (adrianmroz) [reply src/Main.elm:8]: Yup. But first thing we’d need from `Application` are flags for API keys.

Comment (erykpiast) [reply src/Main.elm:8]: Alright. Let's merge this one and create another with displaying some list from API, in simple manner. The key will make sense from the start, then.

**Pull request merged**